packages/@lwc/integration-karma/test/light-dom/slot-fowarding/scoped-slots/index.spec.js

```diff
@@ -0,0 +1,30 @@
+import { createElement } from 'lwc';
+
+import LightContainer from './x/lightContainer/lightContainer';
+
+describe('scoped slots, slot forwarding', () => {
+    let lightContainer;
+    beforeAll(() => {
+        lightContainer = createElement('x-light-container', { is: LightContainer });
+        document.body.appendChild(lightContainer);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(lightContainer);
+    });
+
+    it('does not reassign slot content', () => {
+        const leaf = lightContainer.querySelector('x-leaf');
+        expect(leaf.shadowRoot.children.length).toEqual(2);
+
+        const defaultSlot = leaf.shadowRoot.querySelector('slot');
+        const defaultSlotContent = defaultSlot.assignedNodes();
+        expect(defaultSlotContent.length).toEqual(1);
+        expect(defaultSlotContent[0].innerText).toEqual('Hello world!');
+
+        const fooSlot = leaf.shadowRoot.querySelector('[name="foo"]');
+        const fooSlotContent = fooSlot.assignedNodes();
+        // Note the slot reassignment does not work on scoped slots
+        expect(fooSlotContent.length).toEqual(0);
```

Comment:
Looks like setting the `slot` attribute on the scoped slot doesn't work, added this test to verify the current behavior. 

We can fix this in a separate PR.

Reply:
Recorded a issue for this #3889